core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -22,6 +22,9 @@ import whisk.common.TransactionId
 import whisk.core.entity.Identity
 import whisk.core.loadBalancer.LoadBalancer
 
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Please do not use this import in our components at all. It introduces an independent execution context which does not necessarily relate to the EC in that component. You'll need to pass through the actor system's EC.